iris_pipeline/readout/generate_ramp.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import numpy as np
+
+from jwst.ramp_fitting.ramp_fit import ramp_fit
+from jwst.datamodels import dqflags
+from jwst.datamodels import MIRIRampModel
+from jwst.datamodels import RampModel
+from jwst.datamodels import GainModel, ReadnoiseModel
+
+
+def setup_inputs(ngroups=4, readnoise=10, nints=1,
+                 nrows=4096, ncols=4096, nframes=1, grouptime=1.0,gain=1, deltatime=1):
+        arr=np.array([np.zeros([4096,4096]),np.ones([4096,4096])*4000,np.ones([4096,4096])*8000,np.ones([4096,4096])*12000],dtype=np.float64)
+        arr=np.array([arr])        
+
+        
+        times = np.array(list(range(ngroups)),dtype=np.float64) * deltatime
+        gain = np.ones(shape=(nrows, ncols), dtype=np.float64) * gain
+        err = np.ones(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
+        data = np.zeros(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
+        data=arr
+        pixdq = np.zeros(shape=(nrows, ncols), dtype= np.float64)
+        read_noise = np.full((nrows, ncols), readnoise, dtype=np.float64)
+        gdq = np.zeros(shape=(nints, ngroups, nrows, ncols), dtype=np.int32)
+        model1 = RampModel(data=data, err=err, pixeldq=pixdq, groupdq=gdq, times=times)
+        model1.meta.instrument.name='MIRI'
+        model1.meta.instrument.detector='MIRIMAGE'
```

Comment:
@arunsurya77 both the `test_ramp.fits` file generated by this file and the `sample_ramp.fits` file included in the repo have metadata about MIRI.
Do you have a version of this that instead generates a IRIS ramp?
You should have access to the `import_rop` branch so you can directly commit into this pull request.

Reply:
the generate_ramp.py is actually an oldfile that I used sometime before that is obsolete. I forgot to add the test_ramp.fits  file. i will upload it in figshare and send the link

Reply:
I guess you must have an updated `generate_ramp.py` that you used to create the new `test_ramp.fits`, can you please add it to the repo as well?